docs/components/connectors/out-of-the-box-connectors/amazon-comprehend.md

```diff
@@ -74,36 +74,32 @@ For more details on the fields that can be configured during asynchronous execut
 - **Client request token**: A unique identifier for the request. If you do not set the client request token, Amazon Comprehend generates one.
-- **Data access role ARN (mandatory)**: The Amazon Resource Name (ARN) of the IAM role that grants Amazon Comprehend read access to your input data.
-- **Document classifier ARN**: The Amazon Resource Name (ARN) of the document classifier to use to process the job.
-- **Flywheel ARN**: The Amazon Resource Number (ARN) of the flywheel associated with the model to use.
+- **Data Access Role's ARN (mandatory)**: The Amazon Resource Name (ARN) of the IAM role that grants Amazon Comprehend read access to your input data.
+- **Document Classifier's ARN**: The Amazon Resource Name (ARN) of the document classifier to use to process the job.
+- **Flywheel's ARN**: The Amazon Resource Number (ARN) of the flywheel associated with the model to use.
 - **Job name**: The identifier of the job.
-- **Output S3 URI (mandatory)**: The Amazon S3 location where you want to write the output data. For more details, refer to [output data config](https://docs.aws.amazon.com/comprehend/latest/APIReference/API_OutputDataConfig.html).
-- **Output Kms Key Id**: The ID for the AWS Key Management Service (KMS) key that Amazon Comprehend uses to encrypt the output results from an analysis job. For more details, refer to [output data config](https://docs.aws.amazon.com/comprehend/latest/APIReference/API_OutputDataConfig.html).
+- **Output's S3 URI (mandatory)**: The Amazon S3 location where you want to write the output data. For more details, refer to [output data config](https://docs.aws.amazon.com/comprehend/latest/APIReference/API_OutputDataConfig.html).
+- **Outputs KMS Key Id**: The ID for the AWS Key Management Service (KMS) key that Amazon Comprehend uses to encrypt the output results from an analysis job. For more details, refer to [output data config](https://docs.aws.amazon.com/comprehend/latest/APIReference/API_OutputDataConfig.html).
 - **Tags**: Tags to associate with the document classification job. A tag is a key-value pair that adds metadata to a resource used by Amazon Comprehend.
   **Example:**
 
 ```feel
 = {"status": "active"}
 ```
 
-- **Volume Kms Key Id**: ID for the AWS Key Management Service (KMS) key that Amazon Comprehend uses to encrypt data on the storage volume attached to the ML compute instance(s) that process the analysis job.
-- **Security group ids**: The ID number for a security group on an instance of your private VPC. For more details, refer to [security group](https://docs.aws.amazon.com/comprehend/latest/APIReference/API_VpcConfig.html#comprehend-Type-VpcConfig-SecurityGroupIds).
+- **VolumeKmsKeyId**: ID for the AWS Key Management Service (KMS) key that Amazon Comprehend uses to encrypt data on the storage volume attached to the ML compute instance(s) that process the analysis job.
```

Comment:
These fields are exactly as they are referenced in the UI including casing, yes? If so, you have my approval 👍